app/src/main/java/com/example/sabonit/ProductActivity.java

```diff
@@ -231,6 +233,16 @@ public void goToCart(View view) {
     }
 
     public void addProductToCart(View view) {
+        if(currentProductLiters == 0)
+        {
+            Context context = getApplicationContext();
+            CharSequence text = "You can't add 0 liters!";
+            int duration = Toast.LENGTH_LONG;
+
+            Toast toast = Toast.makeText(context, text, duration);
+            toast.show();
+            return;
+        }
```

Comment:
yes, it was strange if someone can add o liters

Reply:
yes you are right